tests/fixtures/require-spread/good/super.js

```diff
@@ -4,6 +4,7 @@ let Blah = Foo.extend({
 
     // odd, but still needed in some cases
     let { _super } = this;
+
```

Comment:
I think that got taken care of by the `max-statements-per-line` rule, so I think we can nuke this.